runtime/Cargo.toml

```diff
@@ -26,6 +26,12 @@ tracing-subscriber = { workspace = true, features = ["fmt", "json", "env-filter"
 opentelemetry = { workspace = true }
 tracing-opentelemetry = { workspace = true }
 
+[features]
+default = []
+iouring = ["dep:io-uring"]
```

Comment:
Excuse my naïvety, this means "enable io-uring if the dependency exists"?

Reply:
It's saying that when the `iouring` feature is used, we use the `io-uring` dependency.

From https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies:

```
[features]
gif = ["dep:gif"]
```

> This means that this dependency will only be included if the gif feature is enabled. The same cfg(feature = "gif") syntax can be used in the code, and the dependency can be enabled just like any feature such as --features gif (see [Command-line feature options](https://doc.rust-lang.org/cargo/reference/features.html#command-line-feature-options) below).